app/assets/images/payment_icons/airteltigo_mobile_money.svg

```diff
@@ -0,0 +1 @@
21 1.57 1.57 0 001.42-.9 2.16 2.16 0 00.08-1.74zm-.84 1.43a.67.67 0 01-.5.44.75.75 0 01-.93-.51 1.39 1.39 0 01-.08-.57 1.18 1.18 0 01.08-.53.68.68 0 01.63-.5.76.76 0 01.75.41 1.42 1.42 0 01.13.53 1.65 1.65 0 01-.08.73zm-3.43 1.25a.77.77 0 00.61-.74v-.67a.41.41 0 00-.35-.37 4.68 4.68 0 00-.53 0v.08a3.78 3.78 0 010 .75c0 .18-.07.26-.25.27a2.17 2.17 0 01-.44 0 .91.91 0 01-.75-.61 1.64 1.64 0 01-.08-.54 1 1 0 011-1.05 1.73 1.73 0 011.17.24l.15.05v-.3a.63.63 0 00-.28-.56 1.08 1.08 0 00-.54-.2 2.32 2.32 0 00-.79 0 1.9 1.9 0 00-1.22.76 1.85 1.85 0 00-.32 1.37 1.64 1.64 0 00.82 1.28 2.22 2.22 0 001.8.24zm-5.61-.08a.62.62 0 00.49-.86h-.1a1.48 1.48 0 01-.69.06.48.48 0 01-.4-.5v-1.4c0-.1 0-.11.11-.11h.51a1 1 0 00.24 0 .4.4 0 00.34-.32 1.46 1.46 0 000-.42h-1.2v-.77a.56.56 0 00-.52.13.94.94 0 00-.29.42 1.37 1.37 0 000 .4v2a2 2 0 000 .48 1 1 0 00.72.79 1.63 1.63 0 00.79.1zm-7.02.59c-.07.22-.15.44-.22.67s-.15.44-.22.66-.15.44-.22.66a1.55 1.55 0 00-.07-.23c-.08-.27-.17-.54-.25-.81l-.29-.86s0-.1-.07-.1h-.49c0 .3-.08.58-.12.87s-.08.57-.11.86a7.9 7.9 0 00-.11.85 1.85 1.85 0 00.47 0c.07-.55.14-1.09.2-1.63a.5.5 0 000 .13c.15.48.31.95.46 1.43 0 .06.05.09.11.09h.36a.12.12 0 00.13-.1c.12-.39.25-.78.38-1.17l.12-.38.21 1.64h.55c-.12-.87-.24-1.72-.36-2.57zm13.31 1.36a2.77 2.77 0 01-2.34-1.24v.39a1.35 1.35 0 00.33.87 2.75 2.75 0 003.91.21 1.42 1.42 0 00.46-.85 3 3 0 000-.52.49.49 0 000-.11 2.88 2.88 0 01-2.36 1.25zm-4.27-3.16V9.92a1.93 1.93 0 000-.24.5.5 0 00-.4-.44 1.63 1.63 0 00-.44 0v2.45a1.28 1.28 0 00.1.47 1 1 0 001.22.51.3.3 0 00.2-.21A.85.85 0 0028 12h-.39a.31.31 0 01-.28-.28 1.49 1.49 0 01-.06-.26zm.34 2.61l-.3 1a.69.69 0 010 .13.69.69 0 00-.05-.13c-.11-.33-.22-.65-.34-1 0 0 0-.1-.07-.1a2.78 2.78 0 00-.5 0 2.07 2.07 0 00.08.2c.19.51.39 1 .58 1.52a.28.28 0 010 .25c0 .05 0 .11-.07.16a.39.39 0 01-.39.25h-.19v.38h.22a.79.79 0 00.72-.28 1.62 1.62 0 00.19-.37c.26-.67.51-1.35.76-2a.35.35 0 000-.12h-.37c-.23.04-.23.04-.27.11zM20.85 14a1.12 1.12 0 00-.6 0 .82.82 0 00-.62.52 1.17 1.17 0 00-.09.45 1.11 1.11 0 00.12.52.85.85 0 00.78.48.9.9 0 00.93-.59 1.19 1.19 0 00.07-.56.89.89 0 00-.59-.82zm0 1.29a.33.33 0 01-.34.25.35.35 0 01-.36-.23 1 1 0 01-.06-.57 1 1 0 010-.17.33.33 0 01.32-.25.36.36 0 01.37.2 1 1 0 01.09.48 1.49 1.49 0 01-.02.25zm2.79-.96a.46.46 0 00-.35-.4.73.73 0 00-.67.13l-.12.09v-.19H22v1.87a2.78 2.78 0 00.5 0 1.09 1.09 0 000-.17v-1a.21.21 0 010-.14.42.42 0 01.3-.2.23.23 0 01.3.2.76.76 0 010 .15v1a.92.92 0 000 .16h.48v-1.3a1.5 1.5 0 00.06-.2zm2.28.67v-.41a.73.73 0 00-.64-.68.91.91 0 00-1 .59 1.22 1.22 0 000 .8.8.8 0 00.56.54 1.36 1.36 0 001-.09s.05 0 .08-.06-.1-.2-.16-.3-.16.07-.24.09a.69.69 0 01-.47 0 .41.41 0 01-.23-.46zm-1.19-.34a.46.46 0 01.19-.35.33.33 0 01.36 0 .34.34 0 01.16.34c-.23.02-.44.01-.71.01zm8.88-6.73a1.9 1.9 0 011.55.71 1.92 1.92 0 010-.24.78.78 0 00-.3-.67 1.89 1.89 0 00-1.47-.46 2 2 0 00-1.06.44.88.88 0 00-.31.56.93.93 0 000 .38 2 2 0 011.59-.72zm-6.78.94a.49.49 0 00.51-.5.51.51 0 00-.51-.51.5.5 0 00-.49.5.49.49 0 00.49.51z" fill="#143e7b"/></svg>
```

Comment:
The build states an error about the `<title>` attribute. It should be `AirtelTigo Money` instead of `Airtel Tigo Money`